.circleci/config.yml

```diff
@@ -193,7 +194,8 @@ jobs:
             mv repo/pool/main/t/tor/*.deb core/focal/
             git add core/focal/*.deb
             # If there are changes, diff-index will fail, so we commit and push
-            git diff-index --quiet HEAD || git commit -m "Automatically updating Tor packages" && git push origin main
+            git diff-index --quiet HEAD || git commit -m "Automatically updating Tor packages" \
```

Comment:
This did not work like I expected (and already hasn't been working like I expected).

```
[user@dev ~]$ false || echo "1" && echo "2" && echo "3"
1
2
3
[user@dev ~]$ true || echo "1" && echo "2" && echo "3"
2
3
```

Or with parenthesis, I assumed/wanted `diff-index || (commit && push && new-tor-issue)` but instead we got `(diff-index || commit) && push && new-tor-issue`.

Which means it'll always run the script, regardless of whether there's a new package :<